### PR TITLE
Fixes #8763

### LINKED ISSUE #8763: Libnd4j build for android on windows under msys2
#### Issue Description
Output here:
https://gist.github.com/agibsonccc/30c61b60b3d887d4253aa03955f4bd9d

It seems to use the wrong toolchain. It attempts to use MSVC
despite  having the  NDK home and platform set.

Tried with:
```bash
export OPENBLAS_PATH =/c/Users/agibs/.javacpp/cache/openblas-0.3.8-1.5.3-SNAPSHOT-android-arm64.jar/lib/arm64-v8a
./buildnativeoperations.sh -platform android-arm64
```

where the open blas path is set and using maven 
```bash
mvn -Djavacpp.platform=android-arm64 clean install -Dmaven.test.skip=true
```

### PR DESCRIPTION
**Pull request by agibsonccc**
## What changes were proposed in this pull request?
Fixes #8763. Cleans up android and arm running on jetson nano compilation.
(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

## Quick checklist

The following checklist helps ensure your PR is complete:

- [ ] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [ ] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.

### CODE REVIEW

File: libnd4j/CMakeLists.txt [hunk 1]
```diff
@@ -79,7 +79,7 @@ if(NOT SD_CUDA)
     if ("${OPENBLAS_PATH}" STREQUAL "")
         #we don't want OpenBLAS on Apple
         if (NOT APPLE)
-            set(BLA_VENDOR "OpenBLAS")
+            set(BLAS_VENDOR "OpenBLAS")
```
Comment (raver119) [inline libnd4j/CMakeLists.txt]: That's wrong. It's BLA_VENDOR.

**Review by raver119 — DISMISSED** [review]
:)

File: libnd4j/buildnativeoperations.sh [hunk 2]
```diff
@@ -537,7 +589,8 @@ echo CHECK_VECTORIZATION = "$CHECK_VECTORIZATION"
 echo HELPERS = "$HELPERS"
 mkbuilddir
 pwd
-eval $CMAKE_COMMAND  "$BLAS_ARG" "$ARCH_ARG" "$NAME_ARG" -DSD_CHECK_VECTORIZATION="${CHECK_VECTORIZATION}"  $HELPERS "$SHARED_LIBS_ARG" "$MINIFIER_ARG" "$OPERATIONS_ARG" "$BUILD_TYPE" "$PACKAGING_ARG" "$EXPERIMENTAL_ARG" "$TESTS_ARG" "$CUDA_COMPUTE" -DOPENBLAS_PATH="$OPENBLAS_PATH" -DDEV=FALSE -DCMAKE_NEED_RESPONSE=YES -DMKL_MULTI_THREADED=TRUE ../..
+echo "CMAKE COMMAND IS $CMAKE_COMMAND"
```
Comment (raver119) [inline libnd4j/buildnativeoperations.sh]: Do we really need this when merged?

File: libnd4j/buildnativeoperations.sh [hunk 3]
```diff
@@ -154,6 +177,8 @@ if [ "$(uname)" == "Darwin" ]; then
 elif [ "$(expr substr $(uname -s) 1 5)" == "MINGW" ] || [ "$(expr substr $(uname -s) 1 4)" == "MSYS" ]; then
     HOST="windows"
     KERNEL="windows-x86_64"
+    # need to set build path separator, it ends up being wrong on msys2
+    BUILD_PATH_SEPARATOR=";"
```
Comment (saudet) [inline libnd4j/buildnativeoperations.sh:185]: We can remove this. I fixed that in commit https://github.com/bytedeco/javacpp/commit/881ce8e7629488dbb1a9ee4433c13a5b76c231f1

**Pull request merged**